generator/src/command/apiHour.js

```diff
@@ -13,9 +13,9 @@ module.exports = async () => {
   const talkAnswers = [];
 
   let newTalk = false;
-  const today = DateTime.local();
-  today.setLocale("fr");
-  const printedDate = today.toFormat("yyyy-LL-dd");
+  const AHdate = DateTime.fromJSDate(answers.date);
```

Comment:
I don't recognize this case? :trollface: 

Why not calling this variable `eventDate` instead?

Reply:
mainly because this script is specific for api hours, but PRs are welcome if you think `eventDate` is better :kissing:

Reply:
:clap: